<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="1" column="0">
     <widget class="QTextBrowser" name="textBrowser"/>
    </item>
    <item row="0" column="0">
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QLineEdit" name="editsearch"/>
      </item>
      <item>
       <widget class="QPushButton" name="buttonfind">
        <property name="text">
         <string>Поиск</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item row="2" column="0">
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <widget class="QLabel" name="label">
        <property name="text">
         <string> ID:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="editID"/>
      </item>
      <item>
       <widget class="QLabel" name="label_2">
        <property name="text">
         <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:8pt;&quot;&gt;Предмет:&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="editPredmet"/>
      </item>
      <item>
       <widget class="QLabel" name="label_3">
        <property name="text">
         <string>Тип занятия:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QComboBox" name="comboTipe">
        <item>
         <property name="text">
          <string>Лекция</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Праактическое</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Лабораторная</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Экзамен</string>
         </property>
        </item>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_4">
        <property name="text">
         <string>Аудитория:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="editAud"/>
      </item>
      <item>
       <widget class="QPushButton" name="buttonAdd">
        <property name="text">
         <string>Добавить</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonfind</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>findOccupation()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>749</x>
     <y>41</y>
    </hint>
    <hint type="destinationlabel">
     <x>633</x>
     <y>29</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonAdd</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>addOccupation()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>749</x>
     <y>555</y>
    </hint>
    <hint type="destinationlabel">
     <x>399</x>
     <y>299</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>findOccupation()</slot>
  <slot>addOccupation()</slot>
 </slots>
</ui>
